Tools/CISupport/ews-build/steps.py

```diff
@@ -2499,14 +2499,6 @@ def getResultSummary(self):
 
     @defer.inlineCallbacks
     def checkPRStatus(self, pr_number):
-        passed_status_check = self.getProperty('passed_status_check')
```

Comment:
Note: The following lines are deleted since they were artifacts of a different implementation that somehow persisted.